recipes/xarray-behave/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set python_min = "3.10" %}
+
+package:
+  name: xarray-behave
+  version: 0.36.0
+
+source:
+  url: https://files.pythonhosted.org/packages/91/42/b4416c675867e93ecf227c02bcb16aa0bfb358cf429bdf2f3fddef0c9a61/xarray_behave-0.36.0.tar.gz
```

Comment:
Please use grayskull to generate Python recipes. Grayskull make it easier for us to review.

See https://github.com/conda-forge/staged-recipes?tab=readme-ov-file#grayskull---recipe-generator-for-python-packages-on-pypi for more information.